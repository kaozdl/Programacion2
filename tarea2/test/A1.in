insertar_lista .
insertar_lista (1,sol) (2,mercurio) (3,venus) (4,tierra) .
recorrido
insertar_lista (5,marte) . 
recorrido
Fin
