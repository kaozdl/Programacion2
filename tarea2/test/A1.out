1>(0,primero)
2>(0,primero)(0,primero)(1,sol)(2,mercurio)(3,venus)(4,tierra)
3>(0,primero)(0,primero)(1,sol)(2,mercurio)(3,venus)(4,tierra)(4,tierra)(3,venus)(2,mercurio)(1,sol)(0,primero)(0,primero)
4>(0,primero)(0,primero)(0,primero)(1,sol)(2,mercurio)(3,venus)(4,tierra)(5,marte)
5>(0,primero)(0,primero)(0,primero)(1,sol)(2,mercurio)(3,venus)(4,tierra)(5,marte)(5,marte)(4,tierra)(3,venus)(2,mercurio)(1,sol)(0,primero)(0,primero)(0,primero)
6>Fin.
